src/main/java/com/addthis/meshy/MeshyServer.java

```diff
@@ -77,6 +78,10 @@
     private static final boolean allowPeerLocal = Parameter.boolValue("meshy.peer.local", true);
     private static final int autoMeshTimeout = Parameter.intValue("meshy.autoMeshTimeout", 60000);
 
+    private static final int meshyUnitDelay = Parameter.intValue("mss.mesh.unitDelay", 900);
+    private static final int meshyMaxDelayUnit = Parameter.intValue("mss.mesh.maxDelayUnit", 3);
+    private static final int meshyMaxRetries = Parameter.intValue("mss.mesh.maxRetries", 5);
```

Comment:
The parameter names and the variable names here do not make it obvious at all that they are for connection retries. We should make it more obvious with better names.